copter/source/docs/tuning-process-instructions.rst

```diff
@@ -15,50 +15,63 @@ The following parameters should be set correctly based on the specifications of
 
 Battery setting
 ^^^^^^^^^^^^^^^
-Those values a necessary to scale the motor thrust linearisation.
+Parameters used to linearise your motor thrust curve.
 
 - :ref:`MOT_BAT_VOLT_MAX <MOT_BAT_VOLT_MAX>` - 4.2v x No. Cells
 - :ref:`MOT_BAT_VOLT_MIN <MOT_BAT_VOLT_MIN>` - 3.3v x No. Cells
-
-Motors setup
-^^^^^^^^^^^^
-Those values are necessary to scale correctly your motor control.
-
-- :ref:`MOT_PWM_MAX <MOT_PWM_MAX>` - Check ESC manual for fixed range or 2000us
-- :ref:`MOT_PWM_MIN <MOT_PWM_MIN>` - Check ESC manual for fixed range or 1000us
-- :ref:`MOT_SPIN_ARM <MOT_SPIN_ARM>` - use the motor test feature :ref:`Checking the motor numbering with the Mission Planner Motor test`
-- :ref:`MOT_SPIN_MAX <MOT_SPIN_MAX>` - 0.95  # TODO why this value? give a little headroom on ESC?
-- :ref:`MOT_SPIN_MIN <MOT_SPIN_MIN>` - use the motor test feature :ref:`Checking the motor numbering with the Mission Planner Motor test` # TODO : how do select value betwen ARM and MIN ?
 - :ref:`MOT_THST_EXPO <MOT_THST_EXPO>` - 0.55 for 5 inch props, 0.65 for 10 inch props, 0.75 for 20 inch props. This parameter should be derived by thrust stand measurements for best results (don’t trust manufacture data).
-- :ref:`MOT_THST_HOVER <MOT_THST_HOVER>` - 0.25 or below the expected hover thrust percentage (low is safe)
 
 .. plot::
 
     import matplotlib.pyplot as plt
+    import numpy as np
```

Comment:
I guess this part is actually building a graph on the web page?  If it's a static view I think we should just turn it into an image..

Reply:
> I'll leave the decision to @lthall though.

This is really not one of those decisions that I should be making :)

All I can say is what we have here is better than what we have at the moment. Eventually I would like to get better thrust data and refine these graphs. Then I would be generating them in matlab and probably outputting an image file.

My knee jerk reaction is to use an image but as I said, I am not the person to make that decision.

Reply:
ok, I'd prefer an image then unless there are strong objections..

Reply:
My idea with this was to not lost the source. I spend some time to produce those plot and I don't want that somebody updating after spend time on this. 
Numpy and matplotlib are pretty commun include in python, so that shouldn't be the blocking point.
Maybe a solution when a plot is provide would be to ask the source on a gist or some place in the PR. That way we won't lose the source ?

Reply:
@khancyr, if you really want it generated then I'm ok with it..